nft_ingester/src/bin/api/main.rs

```diff
@@ -84,24 +86,39 @@ pub async fn main() -> Result<(), IngesterError> {
         .rocks_db_path_container
         .clone()
         .unwrap_or(DEFAULT_ROCKSDB_PATH.to_string());
-    let secondary_storage_path = config
+
+    let secondary_storage_first_path = config
         .rocks_db_secondary_path_container
         .clone()
-        .unwrap_or(DEFAULT_SECONDARY_ROCKSDB_PATH.to_string());
-    let storage = Storage::open_secondary(
+        .unwrap_or(DEFAULT_SECONDARY_FIRST_ROCKSDB_PATH.to_string());
+    let secondary_storage_first = Storage::open_secondary(
         &primary_storage_path,
-        &secondary_storage_path,
+        &secondary_storage_first_path,
         mutexed_tasks.clone(),
         red_metrics.clone(),
         MigrationState::Last,
     )
     .unwrap();
 
-    let rocks_storage = Arc::new(storage);
+    let secondary_storage_second_path = config
+        .rocks_db_secondary_path_container
```

Comment:
We need to use another env. Now you are using `config.rocks_db_secondary_path_container` for both `secondary_storage_second_path` and `secondary_storage_first_path` but they must be different